packages/gatsby-theme-learning/src/hooks/use-course-status.ts

```diff
@@ -0,0 +1,59 @@
+import useSWR from 'swr';
+import { useContext, useEffect, useState } from 'react';
+import ConfigContext from '../components/config-context';
+import { useAuth0 } from '@auth0/auth0-react';
+import { useAuthToken } from './use-auth-token';
+import type { Course } from '../external-types';
```

Comment:
@FFawzy we need to review the Course type. The type is fine but the API return it wrapped in `result.enrolledCourses` nodes

Reply:
ahah, thank you I'll update the types